shared/templates/sudo_defaults_option/ansible.template

```diff
@@ -8,7 +8,7 @@
 - name: Ensure {{{ OPTION }}} is enabled with the appropriate value in /etc/sudoers
   lineinfile:
     path: /etc/sudoers
-    regexp: '^[\s]*Defaults\s(.*)\b{{{ OPTION }}}=\w+\b(.*)$'
+    regexp: '^[\s]*Defaults\s(.*)\b{{{ OPTION }}}=[-]?\w+\b(.*)$'
```

Comment:
Ah we might need to re-evaluate our `OPTION_REGEX_SUFFIX` then. 

Maybe it should be replaced with `OPTION_VALUE_REGEX` and `OPTION_SUFFIX_REGEX`? Then `OPTION_REGEX_SUFFIX = OPTION_VALUE_REGEX + OPTION_SUFFIX_REGEX` and in bash/ansible we can add-in capture group (around the `OPTION_SUFFIX_REGEX`)? 

Thoughts?

Reply:
I'll see what I can do... I'm probably gonna get the shortcut though xD